.github/workflows/fluent_linter.yaml

```diff
@@ -0,0 +1,32 @@
+name: Lint Fluent Reference Files
+on:
+  push:
+    paths:
+      - 'l10n/en-US/**.ftl'
+      - '.github/workflows/reference_linter.yaml'
```

Comment:
Here, and below, this needs to be updated to account for the renamed file.

Reply:
Good catch 🤦🏼

Reply:
Going to add changes to the linter config to trigger this workflow.